.travis.yml

```diff
@@ -0,0 +1,25 @@
+dist: trusty
```

Comment:
I'd rather use xenial. Trusty is out of support.

Ideally, it should be "bionic" but Travis still don't have the images :-/

Reply:
Ok. Will fix it, just a copy from another repo :)